src/main/java/com/hyperwallet/clientsdk/util/Request.java

```diff
@@ -1,22 +1,16 @@
 package com.hyperwallet.clientsdk.util;
 
-import cc.protea.util.http.Response;
 import cc.protea.util.http.BinaryResponse;
-import java.io.BufferedReader;
-import java.io.ByteArrayOutputStream;
-import java.io.IOException;
-import java.io.InputStream;
-import java.io.InputStreamReader;
-import java.io.OutputStreamWriter;
-import java.lang.reflect.Field;
-import java.lang.reflect.Modifier;
+import cc.protea.util.http.Response;
+import com.hyperwallet.clientsdk.util.Multipart.MultipartData;
```

Comment:
what's the reason of merging normal request object with `MultipartRequest` which is used for Document/File upload?

Reply:
Good question, I'm thinking in move this change to a new PR, but on introducing timeout I saw most of the MultipartRequest is duplicated from Request. The only missing part for Request is writing the multipart into the connection output steam. 

Another benefit is having a single method to prepare the request with HTTP hearders and connections timeout.

Reply:
Oh is this better implemented with Inheritance?, e.g. `MultipartRequest` <extends> `Request`